isatty_others.go

```diff
@@ -1,5 +1,7 @@
 // +build !windows
 // +build !appengine
+// +build !ppc64
+// +build !ppc64le
```

Comment:
Hi, still don't know why you want to exclude `IsCygwinTerminal` function on ppc64 and ppc64le. I think this file doesn't need change.

Reply:
I was confused.